stable/xwiki/templates/statefulset.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: apps/v1beta2
+kind: StatefulSet
+metadata:
+  name: {{ include "xwiki.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "xwiki.name" . }}
+    helm.sh/chart: {{ include "xwiki.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  # without shared storage, no more than 1 replica, also we can't start the next
+  # before killing the old due to the volume attachment. I think.
+  # anyway, safer this way.
+  replicas: 1
+  strategy:
+    type: Recreate
```

Comment:
This can only be `RollingUpdate` or `onDelete`

Reply:
true ... not sure where I got that.